frontend/public/components/dashboard/dashboards-page/cluster-dashboard/getting-started/explore-admin-features-getting-started-card.tsx

```diff
@@ -2,35 +2,21 @@ import * as React from 'react';
 import * as semver from 'semver';
 import { useTranslation } from 'react-i18next';
 import { FlagIcon } from '@patternfly/react-icons/dist/esm/icons/flag-icon';
+import { FLAGS, useOpenShiftVersion } from '@console/shared';
+import { useFlag } from '@console/shared/src/hooks/flag';
 
-import { useOpenShiftVersion } from '@console/shared/src';
 import {
   GettingStartedCard,
   GettingStartedLink,
 } from '@console/shared/src/components/getting-started';
+import { lightspeedOperatorURL } from '@console/app/src/components/lightspeed/Lightspeed';
 import { DOC_URL_OPENSHIFT_WHATS_NEW } from '../../../../utils';
-import { k8sGetResource } from '@console/dynamic-plugin-sdk/src/utils/k8s';
-import { PackageManifestModel } from '@console/operator-lifecycle-manager/src';
-
-const getLightSpeedAvailability = async () => {
-  try {
-    await k8sGetResource({
-      model: PackageManifestModel,
-      name: 'lightspeed-operator',
-      ns: 'openshift-marketplace',
-    });
-    return true;
-  } catch (err) {
-    if (err.response.status !== 404) {
-      // eslint-disable-next-line no-console
-      console.log(err.message);
-    }
-    return false;
-  }
-};
 
 export const ExploreAdminFeaturesGettingStartedCard: React.FC = () => {
   const { t } = useTranslation();
+  const canListPackageManifest = useFlag(FLAGS.CAN_LIST_PACKAGE_MANIFEST);
```

Comment:
why do we need these checks ?

Reply:
It's just for good measure since the user needs to be able to list `PackageManifests` and `OperatorGroups` in order to install a plugin.

Reply:
Right, I forgot that there is the user flows for the button and the getting started card are a bit different 👍